tests/test_llms.py

```diff
@@ -65,6 +67,26 @@ async def ac(x) -> None:
         )
         assert completion.cost > 0
 
+    @pytest.mark.vcr
+    @pytest.mark.asyncio
+    async def test_max_token_truncation(self) -> None:
+        llm = LiteLLMModel(
+            name="gpt-4o-mini",
+            config={
+                "model_list": [
+                    {
+                        "model_name": "gpt-4o-mini",
+                        "litellm_params": {"model": "gpt-4o-mini", "max_tokens": 3},
+                    }
+                ]
+            },
+        )
+        chunk = await llm.acomplete("Please tell me a story")  # type: ignore[call-arg]
```

Comment:
@mskarlin it looks like the `rate_limited` decorator breaks client typing as well. We should fix this at some point